### DISCUSSION TITLE
Hello! Test question: Pasta

### DISCUSSION THREAD

**Discussion by jaimiebutton** [thebuttonclan/pangolin#262] [Q&A]
If you were pasta, what kind/shape would you be?

**Comment by wenzowski** [comment]
![Touched_by_His_Noodly_Appendage_HD](https://user-images.githubusercontent.com/597920/120366599-b4585b00-c2c4-11eb-9992-1bc329208278.jpg)

**Reply by jaimiebutton** [reply]
@wenzowski I could see it
